py-polars/polars/dataframe/frame.py

```diff
@@ -3546,22 +3574,15 @@ def write_delta(
         ArrowInvalidError
             If the DataFrame contains data types that could not be cast to their
             primitive type.
+        TableNotFoundError
```

Comment:
If this can trigger this error, there must be a test raising this error.

Reply:
Done!